virttest/utils_hotplug.py

```diff
@@ -483,3 +485,41 @@ def vcpuhotunplug_unsupport_str():
         return "not currently supported"
     else:
         return ""
+
+
+def create_mem_xml(params):
+    """
+    Create memory device xml.
+    """
+    mem_xml = memory.Memory()
+    mem_model = params.get("mem_model", "dimm")
+    mem_xml.mem_model = mem_model
```

Comment:
we can avoid using `mem_model` variable,
`mem_xml.mem_model = params.get("mem_model", "dimm")`

Reply:
Removed the variable, and assigned it directly.